bigc/resources/orders.py

```diff
@@ -50,6 +50,23 @@ def get_product(self, order_id: int, product_id: int) -> dict:
         """Get a specific order product in an order by ID"""
         return self._api.v2.get(f'/orders/{order_id}/products/{product_id}')
 
+    def get_refund_quote(self, order_id: int, items: list, **kwargs) -> dict:
```

Comment:
We should not allow `items` to be passed as a positional argument since it's used in the body of the request
```suggestion
    def get_refund_quote(self, order_id: int, *, items: list, **kwargs) -> dict:
```